lib/solarwinds_apm/opentelemetry/otlp_processor.rb

```diff
@@ -100,6 +100,7 @@ def init_metrics
       def record_request_metrics(span)
         meter_attrs = meter_attributes(span)
         span_time = calculate_span_time(start_time: span.start_timestamp, end_time: span.end_timestamp)
+        span_time = (span_time / 1e3).round
```

Comment:
OTel span timestamps are in nanoseconds, so the division should be by 1e6 to convert to milliseconds.

Reply:
otlp_processor inherit the function `calculate_span_time` from [solarwinds_processor](https://github.com/solarwinds/apm-ruby/blob/main/lib/solarwinds_apm/opentelemetry/solarwinds_processor.rb#L167), so it's already divided by 1e3.